HHS/Head-Start-TTADP
# [Prod] Reopen select closed FEI goals, add statistics widget to training report dashboard

## Description of change
- Reopen closed FEI goals per user request
- Add statistics widget to training report (frontend) and adjust scopes per discussion to encompass completed sessions, not just completed reports

## How to test
- View the regional dashboard as a user with appropriate permissions and confirm that the TR statistics are correct
- No test for reopening closed FEI goals, it is a data-only migration

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2508
* https://jira.acf.gov/browse/TTAHUB-2780
* https://jira.acf.gov/browse/TTAHUB-2786

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
